trotters/trotters.py

```diff
@@ -0,0 +1,259 @@
+import logging
+import hashlib
+import asyncio
+import discord
+from datetime import datetime
+import time
+
+from redbot.core import checks, commands, Config, modlog
+from redbot.core.bot import Red
+from redbot.core.config import Group
+from redbot.core.commands import Context, Cog
+from redbot.core.utils.predicates import MessagePredicate
+
+try:
+    from redbot.core.commands import GuildContext
+except ImportError:
+    from redbot.core.commands import Context as GuildContext
+
+
+class Trotters(commands.Cog):
```

Comment:
Go ahead and change the name since it can be used with any role. Maybe RoleTracker or something?

Reply:
Just gonna keep it at your comment